main.js

```diff
@@ -0,0 +1,24 @@
+
+//event listener for search button
+document.getElementById("searchButton").addEventListener("click", queryInput);
+
+
+// function to get the query input
+function queryInput (){
+    let input = document.getElementById("input").value;
+    console.log(input)
+}
+
+//send query to server
+
+async function getQuery() {
+    try {
+        const response = await axios.get('https://restcountries.eu/rest/v2/');
```

Comment:
Wat ik vooral bij de andere studenten terug zag komen is dat ze heel veel het object herhalen. Ik wil 'm je ook meegeven:
Wat ik zelf heel vervelend vind is dat ik heel veel `response.data[0]` moet herhalen. Wij developers zijn heel lui, dus is daarom de object spreader in het leven geroepen 😉 
```
const { name: countryName, population, subregion: sbuAreaName, ...otherProps } = response.data[0];
```

_Je hoeft de props niet perse te aliasen_

Reply:
Dag, 
Ik voel me heel slecht over deze opdracht. Ga hem zeker nog afmaken. 
Ik ben de hele dag bezig geweest met  de foutmelding 'axios not registered' . Wat ik ook probeerde, ik kwam daar niet vanaf . Ben daarom niet eens echt aan de opdracht toegekomen. Er was iets waardoor axios niet werkte bij mij. 

Ik zie nu ook dat ik eigenlijk de hele pagina in Javascript had moeten zetten en niet in de html...

Ook daar ben ik niet aan toegekomen. 
In ieder geval bedankt voor je opmerkingen!

grt. Dhiraj